modules/standard/IO.chpl

```diff
@@ -6527,7 +6546,7 @@ proc channel.writef(fmtStr:string, const args ...?k, out error:syserr):bool {
   return false;
 }
 
-// documented in varargs version
+pragma "no doc" // documented in varargs version
```

Comment:
Question about this and the other thing you "no doc"d: are the argument lists sufficiently different to justify both being in the documentation, or could one have been inferred from the other?  I might wait to check with @mppf on those changes